### ISSUE TITLE
Add dark mode by default with optional light mode

### ISSUE THREAD

**Issue by paulshryock** [paulshryock/paul-shryock#71]
For users like myself who prefer to use dark mode when it's available on most modern interfaces, it can be frustrating when there is no dark mode available.

See ["The Quest for the Perfect Dark Mode"](https://www.joshwcomeau.com/react/dark-mode/)

- The user should be able to click a toggle to switch between Light and Dark mode.
- The user's preference should be saved, so that future visits use the correct color theme.
- It should default to the user's "preferred" color scheme, according to their operating system settings. If not set, it should default to Light.
- The site should not flicker on first load, even if the user has selected a non-default color theme.
- The site should never show the wrong toggle state.

![Dark mode flow chart](https://www.joshwcomeau.com/images/dark-mode/flow-chart.png)

This, except default to dark mode instead of light.
